Wilma-Paca/WilmaP-homework7/src/Model.java

```diff
@@ -0,0 +1,164 @@
+import java.util.Iterator;
+import java.util.LinkedList;
+
+public class Model {
```

Comment:
You should have 2 models one to represent a single user and another to manage a list of users